src/app/components/progress-indicator/passive-progress-indicator/index.html

```diff
@@ -0,0 +1,276 @@
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-1.png" />
+              <div class="caption caption-success sky-text-success">Do use passive progress indicators to track sequential steps that other users take.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+    
+        <h3 class="sky-text-danger">Don't use when</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Don't use passive progress indicators for tasks that require user interaction.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-2.png" />
+              <div class="caption caption-danger sky-text-danger">Don't use passive progress indicators when users must take action on any of the steps.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Don’t use passive progress indicators to represent independent steps or statuses that are not sequential.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-3.png" />
+              <div class="caption caption-danger sky-text-danger">Don't use passive progress to track steps that are not sequential.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Don't use passive progress indicators simultaneously for multiple items in lists.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div class="sky-margin-stacked-separate">
+              <img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-4.png" />
+              <div class="caption caption-danger sky-text-danger">Don't display multiple passive progress indicators in lists.</div>
+            </div>
+            <div>
+              <img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-5.png" />
+              <div class="caption caption-success sky-text-success">Do roll up steps into a single status in lists. Use a container such as <a routerLink="/components/flyout">flyouts</a> or <a routerLink="/components/popover">popovers</a> to display details from passive progress indicators.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+      </sky-demo-page-content>
+    
+      <sky-demo-page-content sectionHeading="Anatomy">
```

Comment:
Should the Anatomy section have an entry to call out the pending step? It is formatted differently than the regular step label. It seems like we should have a completed step label entry and a pending step entry since they seem to be as distinct as the completed step action and the active step action in the waterfall indicator anatomy.

Reply:
Updated.